src/integrationTest/kotlin/de/bund/digitalservice/useid/identification/TcTokenControllerIntegrationTest.kt

```diff
@@ -12,7 +12,7 @@ import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.http.MediaType
 import org.springframework.test.web.reactive.server.WebTestClient
-import java.util.*
```

Comment:
I wonder why this has worked at all, because spotless has a rule for "no wildcard imports"

Reply:
Spotless is only used by lefthook. If the lefthooks are not executed, you can commit whatever you want. So if a dev did not configure lefthook to be run automatically for the repo and did not run it manually, noone will stop him from pushing wildcard imports.